x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/flyout.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo, useState } from 'react';
+import { EuiFlyout, EuiFlyoutHeader, EuiTitle, EuiFlyoutBody } from '@elastic/eui';
+import { EuiSpacer, EuiTabs, EuiTab } from '@elastic/eui';
+import { MetadataTab } from './metadata/metadata';
+import type { HostNodeRow } from '../../hooks/use_hosts_table';
+import { useUnifiedSearchContext } from '../../hooks/use_unified_search';
+
+interface Props {
+  node: HostNodeRow;
+  closeFlyout: () => void;
+}
+
+const flyoutTabs = [MetadataTab];
+
+export const Flyout = ({ node, closeFlyout }: Props) => {
+  const { getDateRangeAsTimestamp } = useUnifiedSearchContext();
+
+  const { to, from } = getDateRangeAsTimestamp();
+
+  const currentTimeRange = useMemo(
+    () => ({
+      from,
+      interval: '1m',
+      to,
+    }),
+    [from, to]
+  );
+
+  const tabs = useMemo(() => {
+    return flyoutTabs.map((m) => {
+      const TabContent = m.content;
+      return {
+        ...m,
+        content: <TabContent node={node} currentTimeRange={currentTimeRange} />,
+      };
+    });
+  }, [currentTimeRange, node]);
+
+  const [selectedTab, setSelectedTab] = useState(0);
+
+  return (
+    <EuiFlyout onClose={closeFlyout} ownFocus={false}>
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle size="xs">
+          <h2>{node.name}</h2>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <EuiTabs style={{ marginBottom: '-25px' }} size="s">
```

Comment:
**nit:** should this margin bottom be `16px` as per design? 25px seems an odd number and doesn't respect the scale from our team, use the `euiTheme` spaces to apply this margin to `16px`

Reply:
It'e because we want to match the underline with the tabs line, if it is `16` they won't match: 
<img width="1061" alt="image" src="https://user-images.githubusercontent.com/14139027/226880368-2e07b6d9-1936-42a0-9341-03ebec7900ca.png">